MC/config/PWGDQ/ini/GeneratorHF_bbbar_fwdy.ini

```diff
@@ -0,0 +1,32 @@
+### The setup uses an external event generator
+### This part sets the path of the file and the function call to retrieve it
+
+#[GeneratorExternal]
```

Comment:
why is this line commented?

Reply:
Ciao Roberto, thanks, you are right, I'll fix this in a moment.